providers/class-two-factor-totp.php

```diff
@@ -349,7 +349,7 @@ public function user_two_factor_options( $user ) {
 						/* translators: Example auth code. */
 						$placeholder = sprintf( __( 'eg. %s', 'two-factor' ), '123456' );
 					?>
-					<input type="tel" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9 ]*" placeholder="<?php echo esc_attr( $placeholder ); ?>" />
+					<input type="text" inputmode="numeric" name="two-factor-totp-authcode" id="two-factor-totp-authcode" class="input" value="" size="20" pattern="[0-9 ]*" placeholder="<?php echo esc_attr( $placeholder ); ?>" autocomplete="off" />
```

Comment:
I just confirmed that GitHub is also using `text` input type with `numeric` inputmode:

![totp-input-github](https://github.com/user-attachments/assets/c2369066-bf55-41b9-b357-a5d860c475a4)

But they also have `autocomplete="one-time-code"` ([per spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#one-time-code)), so I wonder if we actually need to change that to match.